### ISSUE TITLE
Oauth Access Token Scopes

### ISSUE THREAD

**Issue by jbb494** [openauthjs/openauthjs#5]
### Feature Request
To add the ability to generate access tokens with Oauth scopes https://datatracker.ietf.org/doc/html/rfc6749#section-3.3

Looks like a simple enough issue, wouldn't mind opening a PR if you believe this project should support it.

**Comment by thdxr** [comment]
yeah this is definitely something we want to support. would definitely take a look at a PR for it but not sure what kind of design questions there are

if you're bored feel free to hack on it and show me

**Comment by lionello** [comment]
@thdxr Did you get a chance to check my PR?
